todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsPresenter.java

```diff
@@ -92,7 +93,7 @@ private void loadStatistics() {
                 })
                 .subscribe(
                         // onNext
-                        stats -> mStatisticsView.showStatistics(stats.first, stats.second),
+                        stats -> mStatisticsView.showStatistics(Ints.saturatedCast(stats.first), Ints.saturatedCast(stats.second)),
```

Comment:
Have you run tests in "prod" as well? You might have to adjust proguard's settings to add some Guava magic.

Reply:
I haven't run them after I made this change, so I'll double check 👍